### ISSUE TITLE
NTR, pulmo: Glycogen accumulation

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6389]
**Preferred term label:**
Glycogen accumulation in (pulmonary?) interstitial cells

**Synonyms**
pulmonary interstitial glycogenosis, glycogen-containing interstitial cells

**Definition (free text, please give PubMed ID)**
accumulation of immature interstitial cells containing abundant cytoplasmic glycogen defined by periodic acid-Schiff (PAS) positive cells follow in interstitial thickening
PMID: 29957271

**Parent term (use hpo.jax.org/app)**
Interstitial pulmonary abnormality HP:0006530

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pulmonary interstitial glycogenosis, ORPHA:217557

**Your nano-attribution (ORCID)**
LMU:chrapp, @kknoflach, @mgriese

**Comment by pnrobinson** [comment]
@GrieseAG  there is a typo in the definition, let's correct that today

**Comment by pnrobinson** [comment]
delete -follow in interstitial thickening

**Comment by pnrobinson** [comment]
added
